### ISSUE TITLE
Check boxes draw differently for staged files

### ISSUE THREAD

**Issue by safield** [gitahead/gitahead#169]
GitAhead v 2.5.3
Windows 10 

When staging a files, some files receive a check in their check box, other files receive a light grey square.

I can not seem to determine what the difference is between the files that receive a check and the files that receive a light grey square.

In addition, the files that receive a check box will unstage when I click the checked box. The staged files that have a light grey box in their check box will not unstage or change in any way when I click their check box. To unstage these files, I have to right click to unstage.

Observe the following screenshot.

![gitahead_stage_bug](https://user-images.githubusercontent.com/12876404/53053525-3f501700-3467-11e9-82b8-e3cca77801c1.jpg)

**Comment by hackhaslam** [comment]
The gray box is supposed to show the intermediate (partially staged) state. Is it showing up for files that are fully staged?

**Comment by safield** [comment]
Yes. I have two files. One drawing a checkmark, and one drawing a grey box. The grey box is persistent beyond refresh and application restart. Calling git status from the command line indicates that the file with the grey box is staged.

**Comment by safield** [comment]
Is there anymore valuable information I could provide on this issue?
